challenges/objects-arrays.js

```diff
@@ -52,19 +75,37 @@ const graduates = [
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+graduates.forEach(univ => {
```

Comment:
Making your variables a little more verbose might help reduce cognitive load for you when working on code.

For example, `graduates.forEach(graduate => {` instead of 'univ' makes it a little easier to understand that the for loop is going through and acting on each graduate in the array 😃